### DISCUSSION TITLE
[Bug]

### DISCUSSION THREAD

**Discussion by Hachi899** [danthe1st/Mee6-bypasser#30] [Support] [answered]
Hi,

Doing !rank doesn't update the roles. I followed the instructions on the previous bug reports but it still doesn't work. The server I am in has approximately 250 and around 20 active members. Do you know what the issue is? If so, is this something that can be fixed?

Thanks.

**Comment by Hachi899** [comment, marked as answer]
Nevermind, it's working now.

**Comment by danthe1st** [comment]
This is a duplicate of #17. The problem is that Mee6-bypasser does not work if too many people try to load roles at the same time.

## [Edit]

This should not be a problem any more.
